src/js/components/Search.jsx

```diff
@@ -24,11 +54,16 @@ export class Search extends PureComponent {
           onSearchChange={(data, searchTerm) => (
             this.handleSearchChange(data, searchTerm)
           )}
+          onSearchClear={() => this.handleSearchChange(null)}
         />
         {/* render results table */}
         {this.state.showResults && <ResultsTable
           {...props}
+          changeColumnWidths={(columns, table) => this.handleCalculateColumnWidths(columns, table)}
```

Comment:
Ah yea I see. In that case can you just add it to propTypes (eslint airBnB rules should be throwing a wobbly that they aren't defined) of <ResultTable>... This should do it:

```
ResultTable.propTypes = {
  ...SussolReactTable.propTypes
  changeColumnWidths: React.PropTypes.func,
};
```